config/faker-generator.js

```diff
@@ -0,0 +1,18 @@
+const _ = require('underscore');
+const fs = require('fs');
+const files = fs.readdirSync('./test/fixtures/config');
+
+_.each(files, (file) => {
+  const entity_factory = require(`../test/fixtures/config/${file}`);
+  const dest = `./test/fixtures/collections/${file.split('.')[0]}.json`;
+
+  const collection = _.times(1000, () => {
```

Comment:
this is...a lot. I bumped it up from 100 to see how long it would take to run, and it was consistently 1 second (either way)

Reply:
I think 100 may be enough.  It may not take more time to generate, but Cypress will slow with lots of data.

Reply:
this could also probably be `_.times(100, entityFactory.generate);`